packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -20,22 +21,28 @@ import {
 
 export enum SourceStatusFlags {
   Local = '--local',
-  Remote = '--remote'
+  Remote = '--remote',
 }
 
 export class ForceSourceStatusExecutor extends SfdxCommandletExecutor<{}> {
+  public command: string;
   private flag: SourceStatusFlags | undefined;
+  private description: string;
+  private logName: string;
 
-  public constructor(flag?: SourceStatusFlags) {
+  public constructor(flag?: SourceStatusFlags, cmdVersion?: CommandVersion) {
     super();
     this.flag = flag;
+    this.command = `force:source:${cmdVersion? cmdVersion : CommandVersion.Default}:status`;
```

Comment:
`cmdVersion ?? CommandVersion.Default` is some TS syntax to make this a little shorter

Reply:
Good to know - thanks. Current refactored code doesn't use this pattern.